src/Note.cpp

```diff
@@ -0,0 +1,378 @@
+/*************************************************************************/
+/*  Xania (M)ulti(U)ser(D)ungeon server source code                      */
+/*  (C) 1995-2000 Xania Development Team                                 */
+/*  See the header to file: merc.h for original code copyrights          */
+/*                                                                       */
+/*  Note.cpp: notes system                                               */
+/*                                                                       */
+/*************************************************************************/
+
+#include "Note.hpp"
+#include "DescriptorList.hpp"
+#include "TimeInfoData.hpp"
+#include "db.h"
+#include "merc.h"
+#include "string_utils.hpp"
+
+#include <fmt/format.h>
+#include <range/v3/algorithm/find_if.hpp>
+#include <range/v3/view/filter.hpp>
+
+#include <cctype>
+#include <cstdio>
+#include <functional>
+
+static const char *NOTE_FILE = "notes.txt";
+
+bool Note::is_to(const Char &ch) const {
```

Comment:
Not that it matters today, but one idea I had is being able to send a note to your whole clan.  Perhaps we'll do that some day if there are players and clans!